config/webpack/build-aot.webpack.config.js

```diff
@@ -19,6 +19,14 @@ function getWebpackConfig(skyPagesConfig) {
   let commonConfig = common.getWebpackConfig(skyPagesConfig);
   commonConfig.entry = null;
 
+  // Since the preloader is executed against the file system during an AoT build,
+  // we need to remove it from the webpack config, otherwise it will get executed twice.
+  commonConfig.module.rules = commonConfig.module.rules
```

Comment:
I like the comment.  Thanks for that.  :-)

Any thoughts on shortening the syntax?  Obviously doing the exact same thing, just removed some extraneous parenthesis and removed the `const`.  I'm pretty impartial on this, just thought the syntax looked a little weird.

```
 commonConfig.module.rules = commonConfig.module.rules
  .filter(rule => !/\/sky-processor\//.test(rule.loader));
```